tests/core/numerics/ohm/:w

```diff
@@ -0,0 +1,643 @@
+
```

Comment:
is the name of this file possibly because of vim? :)

Reply:
seems identical to https://github.com/rochSmets/PHARE/blob/ohm/tests/core/numerics/ohm/test_ohm.py